cmd/titus-imds-cni/env.go

```diff
@@ -106,6 +107,7 @@ func extractEnv(prev *current.Result, pod *v1.Pod) (map[string]string, error) {
 
 	env["TITUS_IAM_ROLE"] = pod.Annotations[mt.IamRoleArnAnnotation]
 	env["X_FORWARDED_FOR_BLOCKING_MODE"] = pod.Annotations[mt.XForwardedForBlockingModeAnnotation]
+	env["TITUS_LOG_IAM_ROLE"] = pod.Annotations[podCommon.AnnotationKeyLogS3WriterIAMRole]
```

Comment:
Is this always set, what if it's blank?

Reply:
If its blank the second iam proxy will not bind, I liked this behavior because if we deploy today nothing changes until this annotation starts getting set by the mutating web hook.

https://github.com/Netflix/titus-executor/pull/522/files#diff-7b7c269c93709bb9c141084f83098abbd1aa53dc7c294a961d5d84a263bfdfccR140

Reply:
Any idea what % of containers will have this set?

Reply:
I believe 100% of containers deployed on the real kubelet.